example-service/src/client.rs

```diff
@@ -38,20 +39,91 @@ async fn main() -> io::Result<()> {
         .parse::<SocketAddr>()
         .unwrap_or_else(|e| panic!(r#"--server_addr value "{}" invalid: {}"#, server_addr, e));
 
-    let name = flags.value_of("name").unwrap().into();
+    let num_connections: usize = flags
```

Comment:
I like the idea of putting the load test behind a flag. That will keep the first experience simpler. Perhaps there can be a server-side load test flag as well, which switches the transport from json to bincode?

Reply:
Alternately we could just have the server listen for both on different ports, with different settings.

Reply:
I like that idea too!